spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/retry/SpringAiRetryAutoConfiguration.java

```diff
@@ -43,7 +43,7 @@
  * @author Christian Tzolov
  */
 @AutoConfiguration
-@ConditionalOnClass(RetryTemplate.class)
+@ConditionalOnClass(TransientAiException.class)
```

Comment:
Wasn't the intention of author to use RetryUtils instead of RetryTemplate? 
The names are similar, and I got confused while reviewing the PR as well 😃